README.md

```diff
@@ -18,21 +18,24 @@ Performs basic site functions in easyengine.
 Runs the site creation.
 
 ```bash
+ee site create example.com                      # install wordpress without any page caching (default)
 ee site create example.com --wp                 # install wordpress without any page caching
 ee site create example.com --wpredis            # install wordpress with page caching
 ee site create example.com --wpsubir            # install wpmu-subdirectory without any page caching
 ee site create example.com --wpsubir --wpredis  # install wpmu-subdirectory with page caching
 ee site create example.com --wpsubdom           # install wpmu-subdomain without any page caching
 ee site create example.com --wpsubdom --wpredis # install wpmu-subdomain with page cache
+
+# Enable SSL using Let’s Encrypt (You can add --letsencrypt along with any other flag.)
+ee site create example.com --letsencrypt
```

Comment:
This is a command that doesn't exist. So do not specify the full command.

If you need to give an example, give a working example.
```ee site create example.com --wpredis --letsencrypt```

Reply:
Also, if we're also supporting `--le`, then mention that too. 
```
[--letsencrypt|--le]
```